This test tries to install cmake from PyPI and cmake needs scikit-build to get
installed so the test tries to install scikit-build too.  The latest release of
scikit-build does not support sunos yet, so this test currently fails.

--- scikit_build-0.17.5/tests/test_setup.py.orig
+++ scikit_build-0.17.5/tests/test_setup.py
@@ -326,6 +326,7 @@
 @pytest.mark.xfail(
     sys.platform.startswith("cygwin"), strict=False, reason="Cygwin needs a release of scikit-build first"
 )
+@pytest.mark.skip(reason="always fails: https://github.com/scikit-build/scikit-build/pull/983")
 def test_setup_requires_keyword_include_cmake(mocker, capsys):
     mock_setup = mocker.patch("skbuild.setuptools_wrap.setuptools.setup")
 
